### ISSUE TITLE
Not working on pop_os 20.04 since last update

### ISSUE THREAD

**Issue by jeffmaier** [nls1729/acme-code#76]
That's it, it stopped working.

Pop_os 20.04
Kernel 5.4.0-7634-generic 
Shell: bash 5.0.16

**Comment by nls1729** [comment]
What stopped working?  The repo contains extension code, Makefiles, etc...

**Comment by nls1729** [comment]
I will try to find out what is not working by asking a few questions.  The issure you opened is asssociated with the github repository acme-code.  The repository contains Gnome Shell Extensions and other supporting files to create zip files to distribute and install the extensions.
Are you having problems with one of the extensions?
Activities Configurator
Do Not Distrub Button
Gnome Shell Extension Reloader
Each extension has a Makefile are you having a problem with a Makefile?
If it is a Makefile, which extensions is associated with the Makefile?
Did you install one of the extensions from https://extensions.gnome.org?
If you are having trouble with an extension installed from the ego website which extension?

**Comment by Esperaunce** [comment]
I'm assuming things here, but I think jeffmaier means Activities Configurator. The latest update seems to have broken the extension, causing it to immediately crash upon installation or starting. There's no work-around for this, it just plain crashes.

**Comment by jeffmaier** [comment]
Yes, I saw some coments on extensions.gnome saying that, but after a reinstall and reboot is working.

**Comment by nls1729** [comment]
@jeffmaier 
@Esperaunce 

I installed Pop_os 20.04 on a test system.  I installed the extension (GS 3.36 version 86) and it works as it should. 
Most extension problems come from conflicts with other extensions.   From some of the comments on the extension website, I suggest you try the following:

Check /usr/share/gnome-shell/extensions and if you find a directory named activities-config@nls1729 remove it.
$ cd  /usr/share/gnome-shell/extensions
$ pwd     to make sure you are where you think you are.
$ rm -rf activities-config@nls1729

Then remove the directory , in your home directory:
$ cd .local/share/gnome-shell/extensions
$ pwd     to make sure you are where you think you are.
$ rm -rf activities-config@nls1729

Shutdown and restart your system.
Go to the ego site extensions.gnome.org
Install Activities Configurator

![Screenshot](https://user-images.githubusercontent.com/2984166/87192316-3a8e3c00-c2c4-11ea-8036-fdf9fa85b6b7.jpg)
